server/authentication/auth.js

```diff
@@ -27,22 +30,22 @@ passport.deserializeUser(function(user, done) {
     done(null, user);
 });
 
-function init(signInService) {
-    const strategy = new LocalStrategy(async (username, password, done) => {
-        try {
-            const user = await signInService.signIn(username, password);
-
-            if (isEmpty(user)) {
-                return done(null, false, {message: 'Incorrect username or password'});
-            }
-
-            return done(null, user);
-        } catch (error) {
-            return done(null, false, {message: 'A system error occured; please try again later'});
-        }
-    });
+function init(signInService, userService, audit, authStrategy) {
```

Comment:
don't need to pass in `config.authStrategy` here as `auth` already has access to `config`